### PR TITLE
Update TBLogger.jl

### PR DESCRIPTION
**Pull request by shashikdm**
minor bug fix for Julia 1.0.3

### CODE REVIEW

File: src/TBLogger.jl [hunk 1]
```diff
@@ -39,7 +39,7 @@ function TBLogger(logdir="tensorboard_logs/run", overwrite=tb_increment;
 	fpath, evfile = create_eventfile(logdir, purge_step, time)
 
     all_files  = Dict(fpath => evfile)
-	start_step = isnothing(purge_step) ? 0 : purge_step
+	start_step = purge_step == nothing ? 0 : purge_step
```
Comment (oxinabox) [inline src/TBLogger.jl]:
```suggestion
	start_step = something(purge_step, 0)
```

Comment (PhilipVinc) [reply src/TBLogger.jl]: I’m amazed by all kinda obscure but useful functions you seem to know.

**Comment by shashikdm** [comment]
Is it possible to do travis-ci testing for multiple versions of julia? for now it seems to work on my machine but in future it would be nice if travis-ci can do it.

**Comment by shashikdm** [comment]
I just realized #19 has better alternative than the latest commit. @PhilipVinc please merge this PR first and then #19.

**Comment by PhilipVinc** [comment]
You are right, it’s a good idea. I will add Travis testing to 1.0 on Wednesday, when I’m back from holidays.

**Pull request merged**